setup_and_run_MBEDTLS_PRIVATE.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
```

Comment:
The conclusion of the team discussion this morning is that the history should be:
1. Preparatory work.
1. Add the scripts. Modify them based on review/CI feedback.
1. Apply the scripts.
1. Subsequent corrections.
1. Optionally, modify the scripts and apply them again.
1. Remove the scripts.

Reply:
Oh, I remember a different conclusion, where the first part of step 5 was moved earlier so that the scripts are only applied once in step 3 (which means rewriting the history at that step), so that it looked:

1. Preparatory work
2. Add the scripts and rework them (multiple commits for multiple rounds or review/CI)
3. Apply the scripts (one commit, re-created every time the scripts are reworked
4. Remove the scripts (can be post-poned until the PR is approved and passes CI) 

I haven't look at  the scripts, so I don't know if they support being re-applied to a codebase where they've already been applied. If they do support that, then I have no objection to the history you suggest.

Can someone else who was in the meeting comment?

Reply:
@mpg yes, the conclusion from the meeting is as you described
I talked with @gilles-peskine-arm afterwards and we agreed that if the script is modified to be idempotent, they it could be safely re-run.

Reply:
We didn't discuss multiple applications in the meeting, just the sequence add-modify-apply, the point being that an application of the script should use the current version of the script and not one from a different commit.

The scripts weren't idempotent when I reviewed them, but close enough that I think it's just a couple of lines to change.